src/sass/edu-benefits.scss

```diff
@@ -311,9 +311,17 @@ form.rjsf {
 .schemaform-back-buttons {
   margin-bottom: 7rem !important;
 }
+
 .schemaform-intro {
   padding-bottom: 1rem;
   margin-bottom: 7rem;
+
+  // To be moved into modules/_m-schemaform.scss when it's merged
+  .process {
```

Comment:
Without shifting the `ol` over to the left again, it created an uncomfortably large space not present in the screenshot:

---
![image](https://cloud.githubusercontent.com/assets/12970166/22892828/b428c81c-f1c8-11e6-8d3c-4271e27f4dad.png)

---
Alternatively, I could have moved the title, text under the title, and buttons over to the right to close the gap, and that would have matched the screenshot referenced, but I didn't think that was intended since it would unalign the content with the banner, breadcrumbs, et al.